app/views/articles/show.html.erb

```diff
@@ -120,15 +166,15 @@
             </div>
           </a>
         <% end %>
-        <h1 class="<%= @article.title_length_classification %>" itemprop="name headline">
+        <h1 class="<%= @article.title_length_classification %>">
           <%= @article.title %>
         </h1>
         <h3>
-          <span itemprop="author" itemscope itemtype="http://schema.org/Person">
-            <meta itemprop="url" content="<%= user_url(@user) %>">
+          <span>
+            <% user_url(@user) %>
```

Comment:
same question here; why not `<%= user_url(@user) %>`. please excuse my ignorance if i'm missing something totally obvious here, though...

Reply:
No worries! See above. 😄

Reply:
Same question/suggestion as above here. As the erb currently is written, this is being invoked in the template but then the output is just being thrown away (not rendered).